packages/cadl-python/test/generated/property-types/models/property/types/models/_models.py

```diff
@@ -0,0 +1,472 @@
+# coding=utf-8
+# pylint: disable=too-many-lines
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+import datetime
+import sys
+from typing import Any, List, Mapping, TYPE_CHECKING, Union, overload
+
+from .. import _model_base
+from .._model_base import rest_field
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from .. import models as _models
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
+
+
+class BooleanProperty(_model_base.Model):
+    """Model with a boolean property.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :ivar property: Property. Required.
+    :vartype property: bool
+    """
+
+    property: bool = rest_field(name="property")
```

Comment:
if we know the rest name is the same as the attribute name, we don't need to include `name="property"` in the `rest_field`

Reply:
done